src/scss/modules/modals.scss

```diff
@@ -19,6 +19,13 @@
   width: 600px;
 }
 
+.modal--form {
+  @extend .modal;
+
+  width: auto;
+  min-width: 544px;
+}
```

Comment:
Use `min-width` instead of `width`.

Then the modal should be set to whatever width is set for `.modal__form-iframe--small` & `.modal__form-iframe--large`.